### PR TITLE
fix(scylla_yaml): change prometheus_address to 0.0.0.0

### PR DESCRIPTION
**Pull request by enaydanov**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [x] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [x] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [x] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by dkropachev — APPROVED** [review]

**Review by bentsi — DISMISSED** [review]

**Comment by bentsi** [comment]
need to fix unit test

**Comment by enaydanov** [comment]
@bentsi, the unit tests were fixed.

File: unit_tests/test_data/test_scylla_yaml_builders/longevity-10gb-3h.result.json [hunk 1]
```diff
@@ -16,6 +16,6 @@
   ],
   "broadcast_rpc_address": "__NODE_IPV6_ADDRESS__",
   "experimental": true,
-  "prometheus_address": "__NODE_IPV6_ADDRESS__",
-  "enable_ipv6_dns_lookup": true
+  "enable_ipv6_dns_lookup": true,
+  "prometheus_address": "0.0.0.0"
```
Comment (aleksbykov) [inline unit_tests/test_data/test_scylla_yaml_builders/longevity-10gb-3h.result.json:20]: @enaydanov should be  here the ipv6 all interface address?

Comment (enaydanov) [reply unit_tests/test_data/test_scylla_yaml_builders/longevity-10gb-3h.result.json:20]: Good question. I think it should.

**Comment by enaydanov** [comment]
Slightly changed the solution and set "0.0.0.0" in another place.

**Review by aleksbykov — APPROVED** [review]
LGTM

**Review by dkropachev — APPROVED** [review]

**Pull request merged**